### DISCUSSION TITLE
darwiniacommunitydao.xyz is out of date and no longer in use

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#151] [Integration]
Reported by @1022ranji 

Affected servcie: https://darwinia-rpc.darwiniacommunitydao.xyz

**Comment by hackfisher** [comment]
Suggest to change to https://darwinia-rpc.dcdao.box/ instead.


https://docs.darwinia.network/build/getting-started/networks/darwinia/


And this: https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.darwinia.network#/explorer

**Comment by boundless-forest** [comment]
- https://github.com/polkadot-js/apps/pull/10682
- https://github.com/darwinia-network/docs/pull/77
- https://github.com/darwinia-network/docs/pull/78
- https://github.com/ethereum-lists/chains/pull/5327
